### ISSUE TITLE
[sros2_cmake] generate_policy fails to provide policies for talker / listener

### ISSUE THREAD

**Issue by mikaelarguedas** [ros2/sros2#89]
## Bug report

**Required Info:**

- Operating System:
  - Ubuntu 18.04
- Installation type:
  - from source
- Version or commit hash:
  - crystal patch2 for all repos
  - master for sros2 (8165d7f10232fe363a1abeaf53d066b400373d5e)
- DDS implementation:
  - Fast-RTPS
- Client library (if applicable):
  - rclpy

#### Steps to reproduce issue

Terminal 1
```
ros2 run demo_nodes_py talker
```
Terminal 2
```
ros2 run demo_nodes_py listener
```
Terminal 3
After the two nodes exchanged multiple messages (~30 seconds)
```
ros2 security generate_policy foo.xml
``` 

#### Expected behavior
`foo.xml` contains policies for the talker and listener nodes, including all default node rules + the chatter topic (publish for talker and subscribe for listener)

 
#### Actual behavior
foo.xml doesnt have any rules for the talker
```xml
<policy version="0.1.0">
  <profiles>
    <profile node="listener" ns="/">
      <services reply="ALLOW">
        <service>~describe_parameters</service>
        <service>~get_parameter_types</service>
        <service>~get_parameters</service>
        <service>~list_parameters</service>
        <service>~set_parameters</service>
        <service>~set_parameters_atomically</service>
      </services>
      <topics subscribe="ALLOW">
        <topic>chatter</topic>
      </topics>
      <topics publish="ALLOW">
        <topic>parameter_events</topic>
        <topic>rosout</topic>
      </topics>
    </profile>
    <profile node="talker" ns="/"/>
  </profiles>
</policy>
```

#### Additional information

`ros2 node info` provides the expected list of topics / services for the `/talker` node
```
# ros2 node info /talker 
/talker
  Subscribers:

  Publishers:
    /chatter: std_msgs/String
    /parameter_events: rcl_interfaces/ParameterEvent
    /rosout: rcl_interfaces/Log
  Services:
    /talker/describe_parameters: rcl_interfaces/DescribeParameters
    /talker/get_parameter_types: rcl_interfaces/GetParameterTypes
    /talker/get_parameters: rcl_interfaces/GetParameters
    /talker/list_parameters: rcl_interfaces/ListParameters
    /talker/set_parameters: rcl_interfaces/SetParameters
    /talker/set_parameters_atomically: rcl_interfaces/SetParametersAtomically
```

@ruffsl @ross-desmond @jacobperron FYI

**Comment by jacobperron** [comment]
I can reproduce the bug.

**Comment by jacobperron** [comment]
Fixed in #91
